xonsh/shell.py

```diff
@@ -60,6 +61,10 @@ def __init__(self, ctx=None, shell_type=None, config=None, rc=None,
                 warn('prompt-toolkit version < v1.0.0 is not supported. '
                      'Please update prompt-toolkit. Using readline instead.')
                 shell_type = 'readline'
+            elif not HAS_PYGMENTS:
+                warn('prompt_toolkit is available but requires pygments.  '
+                     'Please install pygments. Using readline shell')
+                shell_type = 'readline'
```

Comment:
If we have ptk but not pygments, I think we shouldn't need to fall all the way back to readline.  I think we may be able just to turn `$COLOR_INPUT` and `$COLOR_RESULTS` off in this case, and still to use ptk.

That way, we don't have to say that we are falling back to readline, but just that syntax highlighting has been disabled, or something along those lines.